test/images/image-util.sh

```diff
@@ -101,6 +101,21 @@ push() {
   done
 }
 
+# This function will push the manifest list
+push-manifest() {
+  if [[ -f ${IMAGE}/BASEIMAGE ]]; then
+    archs=$(listArchs)
+  else
+    archs=${!QEMUARCHS[@]}
+  fi
+
+  # Make archs list into OS/architecture pair. Eg: 'amd64 ppc64le' to 'linux/amd64,linux/ppc64le'
+  archs=$(echo $archs | sed -e 's/[^ ]* */linux\/&/g' -e 's/ /,/g')
+  TAG=$(<${IMAGE}/VERSION)
+  manifest-tool push from-args --platforms ${archs} --template ${REGISTRY}/${IMAGE}-ARCH:${TAG} --target ${REGISTRY}/${IMAGE}:${TAG}
```

Comment:
why not just fold this call into the `push()` function?

Reply:
@ixdy : Yes we can.  I have updated the PR.